app/models/miq_request_workflow.rb

```diff
@@ -33,6 +33,16 @@ def self.all_encrypted_options_fields
     descendants.flat_map(&:encrypted_options_fields).uniq
   end
 
+  def self.update_requester_from_parameters(data, user)
+    return user if data[:user_name].blank?
+    new_user = User.lookup_by_identity(data[:user_name])
+
+    raise ActiveRecord::RecordNotFound unless new_user
```

Comment:
@mkanoor - Should I create a different log entry along the lines of - `_log.warn "requested not changed to <#{data[:user_name]}> due to a lookup failure"` - Or allow line 42 to log `"requested changed to <nil>"`